### ISSUE TITLE
H2 broken

### ISSUE THREAD

**Issue by mdwragg** [predix-ui/predix-ui.github.io#358]
@evanjd @arcpato @arongraham-ge - something has broken, you're no longer serving over http/2.0. I was using the site to make a point about performance matters and I noted it was slow... Guessing recent changes (netlify?) upset it.

I think an icon is missing also which is causing the service worker registration to blow up.

**Comment by evanjd** [comment]
Hey @mdwragg, thanks for pointing this out. Can you describe in more detail what you're seeing?

From here, I'm seeing no network errors (incl. no missing icon), service workers aren't complaining and [netlify's CDN is serving over HTTP/2](https://http2.pro/check?url=https%3A//predix-ui.com/) (or did you mean preloading assets w/ HTTP/2 push?).

**Comment by mdwragg** [comment]
Seems to be first load is over H2 (but is really slow), second reload is http/1.1. Just running some tracing now, BRB.

**Comment by mdwragg** [comment]
https://www.webpagetest.org/result/181210_H3_c1f9e0ea83ab96291aee93a2b09faabe/

^^ Yeah, these numbers seem off... We were 'A' across the board.

**Comment by mdwragg** [comment]
Ah, yes, ignore my icon thing, starting in a new incognito session seems to have cleared that. Assuming an old SW cached something-or-other. Apologies.

**Comment by evanjd** [comment]
I ran it again twice and got A's for everything except Cache Static Content on both runs: https://www.webpagetest.org/result/181210_6F_5946221b5077e973276cdb450b740656/
https://www.webpagetest.org/result/181210_XD_57f50442859e6c07f5417195351fa88f/

I wonder what's going on.

**Comment by evanjd** [comment]
For what it's worth, I haven't personally noticed any unusually slow performance from my end, though obviously you're much more in tune with what to expect than I would be.

Chrome's performance auditor seems relatively happy, this is with an empty cache:
<img width="729" alt="screen shot 2018-12-10 at 9 07 48 pm" src="https://user-images.githubusercontent.com/4055224/49725399-b0cd5580-fcbf-11e8-85d5-5616d0e211d6.png">

Let me check with @arongraham-ge during his morning to see what he wants to do. It wouldn't be much effort to revert back to the old setup if need be.

Thanks for keeping an eye on us btw! 👍

**Comment by mdwragg** [comment]
Oh wow, those numbers do look much better - how strange!

If you're getting good perf, stick with it. It just looked off when I happened to load up the site to make a point with a colleague ;-)

**Comment by mdwragg** [comment]
<img width="1381" alt="screenshot 2018-12-10 at 11 11 56" src="https://user-images.githubusercontent.com/3594714/49725653-7f776a00-fc6c-11e8-952c-c9202a535365.png">

^^ A screenshot of the second reload in an incognito window

**Comment by evanjd** [comment]
Gah this is weird, I'm getting h2 or spdy for everything, across multiple reloads. I wonder if we're being served by different CDN endpoints.

By the way, when did you notice the crappy performance when you were demoing to a colleague? I only cut over to netlify on your Saturday, prior to that it was still cloudflare + github pages.

**Comment by mdwragg** [comment]
Oh dear, that is odd!

I noticed it this morning (CET, I'm in Europe) so (sadly) after the cut-over by the sound of it.

**Comment by evanjd** [comment]
I tweaked caching configuration on netlify mid Dec and have been (almost) straight A's since. "Compress transfer" is B because netlify doesn't gzip the favicon.

<img width="982" alt="screen shot 2019-02-11 at 9 25 32 am" src="https://user-images.githubusercontent.com/4055224/52540562-ffda5880-2dde-11e9-8be5-1cf8ff8f7889.png">

https://www.webpagetest.org/result/190210_QD_22fa1fce87095bc1c85fc6c7b54de982/

I've hassled a few people in different locations and haven't had any success reproducing the fallback to http/1.1 in Australia, San Ramon or Vancouver.

I'm inclined to close unless there's any objections.
